publish/models/version.py

```diff
@@ -80,6 +83,10 @@ def make_version(cls, dandiset: Dandiset = None) -> str:
     def from_girder(cls, dandiset: Dandiset, client: GirderClient) -> Version:
         draft_folder = client.get_json(f'folder/{dandiset.draft_folder_id}')
 
-        version = Version(dandiset=dandiset, metadata=draft_folder['meta'])
+        metadata = draft_folder['meta']
+        name = metadata['dandiset'].pop('name')
+        description = metadata['dandiset'].pop('description')
```

Comment:
Explicitly check the length here, and throw an exception if it's too long.

Reply:
By the way, the lengths for name and description are in the core scheme.

See https://github.com/dandi/schema/blob/master/scripts/models.py#L163-L167